skyportal/facility_apis/sedmv2.py

```diff
@@ -0,0 +1,233 @@
+            )
+
+        DBSession().add(transaction)
+
+    @staticmethod
+    def delete(request):
+        """Delete a follow-up request from SEDMv2 queue.
+
+        Parameters
+        ----------
+        request: skyportal.models.FollowupRequest
+            The request to delete from the queue and the SkyPortal database.
+        """
+
+        from ..models import DBSession, FollowupRequest, FacilityTransaction
+
+        validate_request_to_sedmv2(request)
+
+        if cfg['app.sedmv2_endpoint'] is not None:
+            altdata = request.allocation.altdata
+
+            req = (
+                DBSession()
+                .query(FollowupRequest)
+                .filter(FollowupRequest.id == request.id)
+                .one()
+            )
+
+            altdata = request.allocation.altdata
+
+            if not altdata:
+                raise ValueError('Missing allocation information.')
+
+            content = req.transactions[0].response["content"]
+            content = json.loads(content)
+
+            uid = content["data"]["id"]
+
+            r = requests.delete(
+                f"{cfg['app.sedmv2_endpoint']}/{uid}",
+                headers={"Authorization": f"token {altdata['api_token']}"},
+            )
+            r.raise_for_status()
+            request.status = "deleted"
+
+            transaction = FacilityTransaction(
+                request=http.serialize_requests_request(r.request),
+                response=http.serialize_requests_response(r),
+                followup_request=request,
+                initiator_id=request.last_modified_by_id,
+            )
+        else:
+            request.status = 'deleted'
+
+            transaction = FacilityTransaction(
+                request=None,
+                response=None,
+                followup_request=request,
+                initiator_id=request.last_modified_by_id,
+            )
+
+        DBSession().add(transaction)
+
+    form_json_schema = {
+        "type": "object",
+        "properties": {
+            "observation_choices": {
+                "type": "array",
+                "title": "Desired Observations",
+                "items": {"type": "string", "enum": ["g", "r", "i", "z", "IFU"]},
+                "uniqueItems": True,
+                "minItems": 1,
+            },
+            "exposure_time": {
+                "title": "Exposure Time [s]",
+                "type": "number",
+                "default": 300.0,
+            },
+            "exposure_counts": {
+                "title": "Exposure Counts",
+                "type": "number",
+                "default": 1,
+            },
+            "maximum_airmass": {
+                "title": "Maximum Airmass (1-3)",
+                "type": "number",
+                "default": 2.0,
+                "minimum": 1,
+                "maximum": 3,
+            },
+            "minimum_lunar_distance": {
+                "title": "Maximum Seeing [arcsec] (0-180)",
```

Comment:
This key and title seem to be about two different properties.

Reply:
Good catch.